crates/rolldown/src/bundler_builder.rs

```diff
@@ -60,6 +64,24 @@ impl BundlerBuilder {
     let file_emitter = Arc::new(FileEmitter::new(Arc::clone(&options)));
 
     apply_inner_plugins(&mut self.plugins);
+    trace_action!(
```

Comment:
This PR is blocked by that a rust bunder instance would be created twice while using CLI, which causes generating two `SessionPreface` event with different `SessionId`.  And I don't find a way good way to deal with this.



@underfin @IWANABETHATGUY cc